AnimateSvg.js

```diff
@@ -0,0 +1,119 @@
+import React, { useRef, useEffect, useState } from 'react';
```

Comment:
Added file to serve as a wrapper for LogoSvg component in order to deliver animation functionality.